src/gather/tests/test_commands.py

```diff
@@ -44,61 +52,78 @@ def fooish():
 @NON_CONFLICTING_COMMANDS.register(name='weird_name')
 @CONFLICTING_COMMANDS.register(name='weird_name')
 def weird_name1():
+    """One of several commands registering for same name"""
     pass
 
 @CONFLICTING_COMMANDS.register(name='weird_name')
 def weird_name2():
+    """One of several commands registering for same name"""
     pass
 
 @CONFLICTING_COMMANDS.register(name='weird_name')
 def weird_name3():
+    """One of several commands registering for same name"""
     pass
 
 class CollectorTest(unittest.TestCase):
 
+    """Tests for collecting plugins"""
+
     def test_collecting(self):
+        """Collecting gives only the registered plugins for given collector"""
         collected = NICE_COMMANDS.collect()
         self.assertIn('main1', collected)
         self.assertIs(collected['main1'], main1)
         self.assertNotIn('baz', collected)
 
     def test_non_collision(self):
+        """Collecting with same name for different collectors does not collide"""
```

Comment:
Collision is when two plugins are registered for the same name. A conflict is when this happens when you collect with the conflict strategy.